quipucords/scanner/network/tests_network_connect_callback.py

```diff
@@ -0,0 +1,149 @@
+#
+# Copyright (c) 2019 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public License,
+# version 3 (GPLv3). There is NO WARRANTY for this software, express or
+# implied, including the implied warranties of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv3
+# along with this software; if not, see
+# https://www.gnu.org/licenses/gpl-3.0.txt.
+#
+"""Test the connect callback capabilities."""
+
+# pylint: disable=ungrouped-imports
+from api.models import (Credential,
+                        ScanTask,
+                        Source)
+
+from django.test import TestCase
+
+from scanner.network.connect import ConnectResultStore
+from scanner.network.connect_callback import ConnectResultCallback
+from scanner.test_util import create_scan_job
+
+
+def build_event(host, return_code=0,
+                stderr=False, msg=False, event=False):
+    """Build event dictionary matching ansible runner response."""
+    event_dict = dict()
```

Comment:
Like above, i think that this code would be more readable if you initially defined the dictionary with the available data inside. For example: 
``` 
res_dict = {'rc': return_code}
if stderr:
    res_dict['stderr'] = stderr
if msg:
    res_dict['msg'] = msg
event_data = {'host': host, 'res': res_dict}
event_dict = {'event': event, 'event_data': event_data}
```